src/sdk/lsf-sdk.js

```diff
@@ -586,11 +586,14 @@ export class LSFWrapper {
         // don't react on duplicated annotations error
         { errorHandler: result => result.status === 409 },
       );
-    }, false, loadNext, exitStream);
```

Comment:
the only place passed this parameter in is this place, so please remove `exiStream` from `submitCurrentAnnotation()` as well.